src/writeRedirect.js

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs'
+import _ from 'lodash'
+
+const writeRedirect = (documentData) => {
+  // Prismic custom field with redirect value
+  const redirectValue = documentData?.redirect?.json?.value
+  // Prismic custom field for slug or UID which every document has by default
+  const pageUrl = documentData?.slug?.json?.value || documentData.uid
+  if (!redirectValue || !pageUrl || !redirectValue.includes('https://')) return
+
+  try {
+    const redirectCommand = `aws s3 cp s3://$BUCKET/${pageUrl}.html s3://$BUCKET/${pageUrl}.html --website-redirect ${redirectValue} \n`
```

Comment:
do we need to also apply the redirect to the non `.html` version due to how paulton's site is setup?

Reply:
Good point.